business/istio_certs.go

```diff
@@ -103,7 +103,7 @@ func (ics *IstioCertsService) getCertificateFromSecret(secretName, certName stri
 func (ics *IstioCertsService) getCertsConfigFromIstioConfigMap() ([]certConfig, error) {
 	cfg := config.Get()
 
-	istioConfigMap, err := ics.k8s.GetConfigMap(cfg.IstioNamespace, IstioConfigMapName(*cfg, ""))
+	istioConfigMap, err := ics.k8s.GetConfigMap(cfg.IstioNamespace, cfg.ExternalServices.Istio.ConfigMapName)
```

Comment:
Why wouldn't we want to keep this common function here and in the common function just return `cfg.ExternalServices.Istio.ConfigMapName`... in the future, if anything needs to change (like trying to get the name automagically) you only have to change this function and everywhere the name needs to be obtained will just work.

With this revert, if we ever have to change this, we need to go through all these places again and change it back to calling a common function.

I would leave the common function here - and for now just have it simply `return config.Get().ExternalServices.Istio.ConfigMapName`

Reply:
> Why wouldn't we want to keep this common function here and in the common function just return cfg.ExternalServices.Istio.ConfigMapName... in the future, if anything needs to change (like trying to get the name automagically) you only have to change this function and everywhere the name needs to be obtained will just work.

In the future, I would expect callers to call `GetMesh` to get the config info from there instead of fetching and unmarshaling the configmap every the config is needed.